camunda.Dockerfile

```diff
@@ -31,8 +31,8 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     --mount=type=cache,target=/var/log/apt,sharing=locked \
     apt-get -qq update && \
-    apt-get install -yqq --no-install-recommends tini ca-certificates && \
-    apt-get upgrade -yqq --no-install-recommends
+    apt-get install -yqq --no-install-recommends tini ca-certificates systemd perl && \
+    apt-get upgrade -yqq --no-install-recommends systemd perl
```

Comment:
🔧 The reason for such a change should be described in the commit message for future git archeology. I foresee others reading the Dockerfiles and wondering why we install these. I strongly recommend writing more comprehensive commit messages.